package.json

```diff
@@ -26,17 +26,17 @@
     "prettier": "^1.16.2",
     "qrcode": "^1.4.4",
     "query-string": "^6.2.0",
-    "react": "^16.7.0",
+    "react": "^16.13.1",
```

Comment:
since you're making some potentially breaking changes, I think we should merge all existing PRs before this one to avoid conflicts

Reply:
Yeah, that's a good idea. What about #247 tho?

Reply:
^ closed it. if we're going to add stats component, we'll need to account for our move to NextJS and that the PR is >1 year old.